src/main/resources/templates/db_scripts/migration/create_banking_information_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS banking_information (
```

Comment:
we don't need this script because RMB will create this table automatically after decalring in schema.json

Reply:
Deleted